sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample9_RecognizeCustomEntitiesAsync.cs

```diff
@@ -51,7 +51,7 @@ public async Task RecognizeCustomEntitiesAsync()
             {
                 RecognizeCustomEntitiesActions = new List<RecognizeCustomEntitiesAction>()
                 {
-                    new RecognizeCustomEntitiesAction(projectName, deploymentName);
+                    new RecognizeCustomEntitiesAction(projectName, deploymentName)
```

Comment:
To avoid this kind of mistake going forward, could we adjust the approach used on L48-L65 to just reassign the variable names when not running as a snippet?   That will ensure that the block of code is compiled and tested as part of nightly runs.

```csharp
            // To train a model to recognize your custom entities, see https://aka.ms/azsdk/textanalytics/customentityrecognition
            string projectName = "<projectName>";
            string deploymentName = "<deploymentName>";

#if !SNIPPET
            projectName = TestEnvironment.RecognizeCustomEntitiesProjectName;
            deploymentName = TestEnvironment.RecognizeCustomEntitiesDeploymentName);
#endif
            var actions = new TextAnalyticsActions()
            {
                RecognizeCustomEntitiesActions = new List<RecognizeCustomEntitiesAction>()
                {
                    new RecognizeCustomEntitiesAction(projectName, deploymentName)
                }
            };
```